src/xmipp/applications/scripts/particle_boxsize/batch_particle_boxsize.py

```diff
@@ -190,7 +190,7 @@ def run(self):
         # Default shifts for cropping the images. TODO: add as arguments?
         shift = np.max([config["img_rows"], 1])
         shifts = np.repeat(shift, 2)
-        feature_scaler = pickle.load(open(feature_scaler, "rb"))
+        feature_scaler = pickle.load(open(feature_scaler, "rb"), encoding='latin1')
```

Comment:
utf-8, maybe?

Reply:
This is a fixup to allow python3 pickle to load python2 pickled objects.